### PR TITLE
[TCD-1960] Caution about AAB files with Emulators

### PR DESCRIPTION
**Pull request by sweeneyskirt-sl**
### Description
Added a note about emulator tests not starting if the target app is an AAB file.

### Motivation and Context
Resolve TCD-1960

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [X] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/CONTRIBUTING.MD) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/mobile-apps/automated-testing/appium/quickstart.md [hunk 1]
```diff
@@ -62,5 +62,9 @@ mvn clean test
 ```
 Alternatively, you can right click the test and choose `Run` from most IDE interfaces.
 
+:::note
+For Android tests, if your emulator session fails to start, make sure the app you are targeting is an `\*.apk`, not an `\*.aab`, as the latter is not supported in emulator tests.
```
Comment (linafarid) [inline docs/mobile-apps/automated-testing/appium/quickstart.md:66]: should we maybe say here:
"... as the latter is not supported in emulator test, yet." ?

**Review by linafarid — COMMENTED** [review]
see my comment - shall we add "yet"?

File: docs/mobile-apps/automated-testing/appium/quickstart.md (hunk 1 above)
Comment (sweeneyskirt-sl) [inline docs/mobile-apps/automated-testing/appium/quickstart.md:66]:
```suggestion
For Android tests, if your emulator session fails to start, make sure the app you are targeting is an `\*.apk`, not an `\*.aab`, as the latter is not yet supported in emulator tests.
```

File: docs/mobile-apps/automated-testing/appium/virtual-devices.md [hunk 2]
```diff
@@ -339,6 +343,10 @@ where you'll find demo scripts for a variety of use cases to get you started wit
 * [Ruby](https://github.com/saucelabs-training/demo-ruby/blob/docs-1.0/appium-examples)
 * [C#](https://github.com/saucelabs-training/demo-csharp)
 
+::note
+For Android tests, if your emulator session fails to start, make sure the app you are targeting is an `\*.apk`, not an `\*.aab`, as the latter is not supported in emulator tests.
```
Comment (sweeneyskirt-sl) [inline docs/mobile-apps/automated-testing/appium/virtual-devices.md:347]:
```suggestion
For Android tests, if your emulator session fails to start, make sure the app you are targeting is an `\*.apk`, not an `\*.aab`, as the latter is not yet supported in emulator tests.
```

File: docs/mobile-apps/automated-testing/espresso-xcuitest.md [hunk 3]
```diff
@@ -59,6 +59,9 @@ Modify the `config.yml` file to run your existing tests.
 * **TestObject Migration:** refer to the [Commands Map](#legacy-to-saucectl-commands-map) to determine which CLI commands and/or YAML configuration properties to use based on your TestObject configurations.
 * **New Accounts:** see the `saucectl` configuration documentation for [Espresso](/testrunner-toolkit/configuration/espresso) and [XCUITest](/testrunner-toolkit/configuration/xcuitest).
 
+For Android tests, if your emulator session fails to start, make sure the app you are targeting is an `\*.apk`, not an `\*.aab`, as the latter is not supported in emulator tests.
```
Comment (sweeneyskirt-sl) [inline docs/mobile-apps/automated-testing/espresso-xcuitest.md:62]:
```suggestion
For Android tests, if your emulator session fails to start, make sure the app you are targeting is an `\*.apk`, not an `\*.aab`, as the latter is not yet supported in emulator tests.
```

File: docs/testrunner-toolkit/running-tests.md [hunk 4]
```diff
@@ -250,3 +250,7 @@ test(testName, async t => {
 
 </TabItem>
 </Tabs>
+
+::note
+For Android tests, if your emulator session fails to start, make sure the app you are targeting is an `\*.apk`, not an `\*.aab`, as the latter is not supported in emulator tests.
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/running-tests.md:255]:
```suggestion
For Android tests, if your emulator session fails to start, make sure the app you are targeting is an `\*.apk`, not an `\*.aab`, as the latter is not yet supported in emulator tests.
```

**Comment by sweeneyskirt-sl** [comment]
@linafarid -- good call. I added "yet" to each of the notes. Kindly re-review.

**Review by linafarid — APPROVED** [review]

**Pull request merged**